engine/src/test/java/net/jqwik/api/ArbitrariesEmailsTests.java

```diff
@@ -116,7 +115,7 @@ void tldNotAllNumeric(@ForAll("emails") String email) {
 			assertThat(stringContainsMinimumOneChar(ALLOWED_NOT_NUMERIC_CHARS_TLD, tld)).isTrue();
 		}
 
-		@Property
+		@Property(tries = 5000)
```

Comment:
Why would you try that 5000 times?

Reply:
To have a greater chance that addresses like a::a are generated